knexfile.js

```diff
@@ -0,0 +1,31 @@
+// Update with your config settings.
+
+module.exports = {
+  development: {
+    client: "sqlite3",
+    connection: {
+      filename: "./data/expat.db3",
+    },
+    useNullAsDefault: true,
+    migrations: {
+      directory: "./migrations",
```

Comment:
Could we put the `migrations` in `/data` as well?
Given that `seeds` is in there it seems like it would make sense for everything in the database to live there.